packages/theme/src/Components/SearchableList/SearchableList.tsx

```diff
@@ -72,7 +72,7 @@ export function SearchableList<T extends {}>({
                 shouldSort: true,
                 threshold: 0.45,
                 minMatchCharLength: 1,
-                keys: searchKey ?? data.length > 0 ? Object.keys(data[0]) : [],
+                keys: searchKey ?? Object.keys(data.length > 0 ? data[0] : []),
```

Comment:
So, they are different, right?

Reply:
Yes, they will cause #6073

Reply:
I see, it's about the operator precedence.